src/app/forms/components/approval-type/approval-type.component.ts

```diff
@@ -6,37 +6,87 @@ import { BaseControlComponent } from '../base-control/base-control.component';
 import { FormNodeEditTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
 
 const patterns: Record<string, RegExp> = {
-  NTA: /^(\w+)$/i,
-  'IVA - DVSA/NI': /^(\w+)$/i,
-  IVA: /^(\w+)$/i,
-  ECTA: /^e(\w{2})\*(\w{4})\/(\w{4})\*(\w{6})$/i,
-  NSSTA: /^e(\w{2})\*NKS\*(\w{6})$/i,
-  ECSSTA: /^e(\w{2})\*KS(\w{2})\/(\w{4})\*(\w{6})$/i,
-  'GB WVTA': /^(\w{3})\*(\w{4})\/(\w{4})\*(\w{7})$/i,
-  'UKNI WVTA': /^(\w+)11\*(\w{4})\/(\w{4})\*(\w{6})$/i,
-  'EU WVTA Pre 23': /^e(\w{2})\*(\w{4})\/(\wt{4})\*(\w{6})$/i,
-  'EU WVTA 23 on': /^e(\w{2})\*(\w{4})\/(\w{4})\*(\w{6})$/i,
-  QNIG: /^e(\w{2})\*(\w{4})\/(\w{4})\*(\w{6})$/i,
-  'Prov.GB WVTA': /^(\w{3})\*(\w{4})\/(\w{4})\*(\w{6})$/i,
-  'Small series': /^(\w+)11\*NKS(\w+\*)?(\w{6})$/i,
-  'IVA - VCA': /^n11\*NIV(\w{2})\/(\w{4})\*(\w{6})$/i
+  NTA: /^(.+)$/i, // 25
+  'IVA - DVSA/NI': /^(.+)$/i, // 25
+  IVA: /^(.+)$/i, //25
+  ECTA: /^e(.{2})\*(.{4})\/(.{4})\*(.{6})$/i, //20
+  NSSTA: /^e(.{2})\*NKS\*(.{6})$/i, //14
+  ECSSTA: /^e(.{2})\*KS(.{2})\/(.{4})\*(.{6})$/i, // 18
+  'GB WVTA': /^(.{3})\*(.{4})\/(.{4})\*(.{7})$/i, // 21
+  'UKNI WVTA': /^(.{1})11\*(.{4})\/(.{4})\*(.{6})$/i, // 20
+  'EU WVTA Pre 23': /^e(.{2})\*(.{4})\/(.{4})\*(.{6})$/i, // 20
+  'EU WVTA 23 on': /^e(.{2})\*(.{4})\/(.{4})\*(.{6})$/i, // 20
+  QNIG: /^e(.{2})\*(.{4})\/(.{4})\*(.{6})$/i, // 20
+  'Prov.GB WVTA': /^(.{3})\*(.{4})\/(.{4})\*(.{6})$/i, // 20
+  'Small series': /^(.)11\*NKS(.{2})\*(.{6})$/i, // 16
+  'IVA - VCA': /^n11\*NIV(.{2})\/(.{4})\*(.{6})$/i // 19
```

Comment:
could these patterns get stored inside of a separate file somewhere instead of the component?